pkg/proxy/ipvs/proxier.go

```diff
@@ -453,7 +454,11 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	builtinModsFilePath := fmt.Sprintf("/lib/modules/%s/modules.builtin", kernelVersion)
 	b, err = ioutil.ReadFile(builtinModsFilePath)
 	if err != nil {
-		glog.Errorf("Failed to read file %s with error %v", builtinModsFilePath, err)
+		if os.IsNotExist(err) {
```

Comment:
@m1093782566 The file doesn't exist when kube-proxy is running inside container without mounting `/lib/modules`. Returning a warning instaed of an error may be more appropriate under this circunstance. like https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/proxier.go#L467